### ISSUE TITLE
Keybase config now under "Application Support" instead of `XDG_CONFIG_HOME/keybase` ?

### ISSUE THREAD

**Issue by jasonkarns** [keybase/keybase-issues#2179]
I previously had the npm-installed keybase CLI and have just switched to the brew-installed CLI. Now the new cli states that the config is under: `/Users/<user>/Library/Application Support/Keybase/config.json` whereas it was previously under `XDG_CONFIG_HOME/keybase/config.json`

Is there a way to tell keybase to prefer the XDG-compliant location?

**Comment by jasonkarns** [comment]
More generally, I would prefer to have public-shareable configs stored separately from private-info config. I'm assuming that the salt should remain private. user.device and user.id can be public?

The main thing is to be able to share settings like `gpg` between machines ala dotfiles. Will keybase merge these two files and respect both? Or is there any way to split the public from private info to make managing via dotfiles easier?

**Comment by cjb** [comment]
No, `$HOME/Library/Application Support` is hardcoded as the location on OS X.

A difference between the npm/go versions is that we now have several different binaries who need to be able to locate each other, so we'd prefer not to allow too much flexibility here.

**Comment by jasonkarns** [comment]
So right now, there's no way to share configs between machines?
